gc.c

```diff
@@ -13916,6 +14171,17 @@ Init_GC(void)
 	rb_include_module(rb_cWeakMap, rb_mEnumerable);
     }
 
+    {
+        VALUE rb_cWeakKeyMap = rb_define_class_under(rb_mObjSpace, "WeakKeyMap", rb_cObject);
+        rb_define_alloc_func(rb_cWeakKeyMap, wkmap_allocate);
+        rb_define_method(rb_cWeakKeyMap, "[]=", wkmap_aset, 2);
+        rb_define_method(rb_cWeakKeyMap, "[]", wkmap_aref, 1);
+        rb_define_method(rb_cWeakKeyMap, "getkey", wkmap_getkey, 1);
```

Comment:
I think it would be more consistent to call this method `get_key`. WDYT?

Reply:
Hum, I don't remember why `getkey` was chosen over `get_key`, but that's what was agreed by Matz: https://bugs.ruby-lang.org/issues/18498